src/common/fs/read-json-sync.injectable.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import fsInjectable from "./fs.injectable";
+
+const readJsonSyncInjectable = getInjectable({
+  id: "read-json-sync",
+  instantiate: (di) => di.inject(fsInjectable).readJsonSync,
```

Comment:
Is this is detached from current `override-fs-with-fakes`? That can throw someone off.

Reply:
I don't understand what you are saying.